packages/builder/src/api.js

```diff
@@ -45,4 +45,15 @@ export const API = createAPIClient({
       }
     }
   },
+  onMigrationDetected: appId => {
+    const updatingUrl = `/builder/app/${appId}/updating`
+
+    if (window.location.pathname === updatingUrl) {
+      return
+    }
+
+    window.location = `${updatingUrl}?returnUrl=${encodeURIComponent(
```

Comment:
We can use routify here can't we? So we could use `goto` rather than doing a hard window.location navigation. `goto` also will handle params for us.
https://www.routify.dev/docs/helpers#goto